parsing/lexer.mll

```diff
@@ -618,12 +617,10 @@ and comment = parse
             error_loc loc (Unterminated_string_in_comment (start, str_start))
         in
         is_in_string := false;
-        store_string_char '\"';
```

Comment:
That's a separate change right ? I'm not sure I understand what it does.

Reply:
That should have stayed.